awx/main/models/unified_jobs.py

```diff
@@ -1137,11 +1137,9 @@ def result_stdout_raw_handle(self, enforce_max_bytes=True):
                     if total > max_supported:
                         raise StdoutMaxBytesExceeded(total, max_supported)
 
-                # psycopg2's copy_expert writes bytes, but callers of this
+                # psycopg3's copy writes bytes, but callers of this
                 # function assume a str-based fd will be returned; decode
                 # .write() calls on the fly to maintain this interface
```

Comment:
but you're not doing what this comment describes anymore now, are you?

Reply:
Done.